app/src/main/java/com/example/aima_id_app/data/model/submodel/Address.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.aima_id_app.data.model.submodel
+
+
+/**
+ * Represents a the address of a user.
+ *
+ * @property street represents the name of the street of the user.
+ * @property number represents the number of the user's residence.
+ * @property city represents the city of the user's residence.
+ * @property postalCode represents the postal code of the given address.
+ */
+class Address (
+    val street: String,
+    val number: Int,
+    val city: String,
+    val postalCode: Int,
```

Comment:
The postalCode should be of type String.

Reply:
done